test/ng/directive/ngSrcsetSpec.js

```diff
@@ -28,5 +28,12 @@ describe('ngSrcset', function() {
     $rootScope.$digest();
     expect(element.attr('srcset')).toBe('http://example.com/image1.png 1x,unsafe:javascript:doEvilStuff() 2x');
   }));
+
+  it('should not throw an error if undefined', inject(function($rootScope, $compile) {
```

Comment:
Yes, you're right. That test wasn't testing the fix.

I've updated the test and verified that it fails without the fix.